src/EditorFeatures/Core/Implementation/Classification/SyntacticClassificationTaggerProvider.cs

```diff
@@ -15,14 +18,17 @@
 
 namespace Microsoft.CodeAnalysis.Editor.Implementation.Classification
 {
-    [Export(typeof(ITaggerProvider))]
+    [Export(typeof(IViewTaggerProvider))]
     [ContentType(ContentTypeNames.CSharpContentType)]
     [ContentType(ContentTypeNames.VisualBasicContentType)]
     [TextViewRole(PredefinedTextViewRoles.Document)]
     [TagType(typeof(IClassificationTag))]
-    internal partial class SyntacticClassificationTaggerProvider : ITaggerProvider
+    internal partial class SyntacticClassificationTaggerProvider : IViewTaggerProvider
```

Comment:
Does this still work with RTF-copy, scroll margin, etc?  I thought the editor only call regular taggers for those, not `IViewTaggerProvider`?